src/ServiceControl/Notifications/Email/SendEmailNotificationHandler.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace ServiceControl.Notifications.Email
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus;
+    using NServiceBus.Logging;
+    using NServiceBus.Transport;
+    using Raven.Client;
+    using ServiceBus.Management.Infrastructure.Settings;
+
+    public class SendEmailNotificationHandler : IHandleMessages<SendEmailNotification>
+    {
+        readonly IDocumentStore store;
+        readonly EmailThrottlingState throttlingState;
+
+        public SendEmailNotificationHandler(IDocumentStore store, Settings settings, EmailThrottlingState throttlingState)
+        {
+            this.store = store;
+            this.throttlingState = throttlingState;
+
+            emailDropFolder = settings.EmailDropFolder;
+        }
+
+        public async Task Handle(SendEmailNotification message, IMessageHandlerContext context)
+        {
+            NotificationsSettings notifications;
+
+            using (var session = store.OpenAsyncSession())
+            {
+                notifications = await session.LoadAsync<NotificationsSettings>(NotificationsSettings.SingleDocumentId)
+                    .ConfigureAwait(false);
+            }
+
+            if (notifications == null || !notifications.Email.Enabled)
+            {
+                log.Info("Skipping email sending. Notifications turned-off.");
```

Comment:
I was going to argue that this should be logged at debug, but this is the case where we have in-flight messages for sending emails and then we turn notifications off, right? if so, that makes sense to log at this level.